packages/block-library/src/latest-posts/style.native.scss

```diff
@@ -27,11 +27,10 @@
 }
 
 .latestPostBlockMessage {
-	margin-top: 4;
 	text-align: center;
-	color: $gray-dark;
+	margin-top: 8;
 	font-size: 14;
-	font-weight: 600;
+	color: #2e4453;
```

Comment:
Curious, why do we hard code this in place of the sass var?

Reply:
We had made some design tweaks and I was aligning to what’s in the media placeholder. That used the hex color instead of a var so I really had just copied over settings and didn’t look up to see if that has a matching car defined.

Reply:
Ah, this one was using the `darken` function, so might have been harder to spot: https://github.com/wordpress-mobile/gutenberg-mobile/blob/7500e0eb4d62279659c3af100c1311d255e099a7/src/_colors.scss#L98

Reply:
Ah yeah that one didn't pop up on my search. I'll make a PR to fix that.